src/app/modules/map/components/map-image/map-image.component.ts

```diff
@@ -127,15 +127,14 @@ export class MapImageComponent implements OnInit, OnDestroy, OnChanges {
     this.ngDestroy$.complete();
   }
 
-  onMapReady(map: MapView) {
-    this.mapView = map;
+  onMapReady(mapComponent: MapComponent) {
     const symbol = MarkerHelper.getIconSvg(this.location.geoHazard);
     const marker = new Graphic({
       geometry: this.centerLocation,
       symbol: symbol
     });
     this.markerLayer.add(marker);
-    this.mapView.map.add(this.markerLayer);
+//TODO: VI trenger ikke dette    this.mapView.map.add(this.markerLayer);
```

Comment:
Ikke helt tydelig for meg hva vi ikke trenger? Kan kodelinje som er kommentert ut bare fjernes?

Reply:
Hehe:) Fjerner denne!